interop/src/server.rs

```diff
@@ -445,3 +448,5 @@ const HOME: &str = r##"
   </body>
 </html>
 "##;
+
+const VERSION: &str = "quinn_h3:0.0.1";
```

Comment:
I think it should be spelled as "quinn-h3" in non-code text. What do you think?

Reply:
Yeah kebab case is better here, didn't pay much attention.